src/data_inclusion/schema/services.py

```diff
@@ -63,6 +62,19 @@ class Service(BaseModel):
     ] = None
     zone_diffusion_nom: Optional[str] = None
     contact_nom_prenom: Optional[str] = None
+    page_web: Annotated[
+        Optional[HttpUrl],
+        Field(
+            description="""
+                Lien vers une page web dédiée au service sur le site web de la
+                structure. Ce champ n'est pas destiné à recevoir un lien vers le site
+                d'un producteur de donnée.
+            """,
+            examples=[
+                "https://insersol.fr/biclou-atelier-reparation-participatif-solidaire/"
+            ],
+        ),
+    ] = None
```

Comment:
@Caldagues tu peux voir ici la façon de procéder pour décrire un champ du schéma (markdown autorisé), si tu veux en faire un peu

Je pense qu'on pourrait même avoir une description double :

```suggestion
    page_web: Annotated[
        Optional[HttpUrl],
        Field(
            description="""
                Lien vers une page web dédiée au service sur le site web de la
                structure.

                ## Pour les producteurs :
                
                Ce champ n'est pas destiné à recevoir un lien vers le site
                d'un producteur de donnée.
                
                ## Pour les consommateurs :
                
                 Ce champ doit être utilisé lorsque que le mode d'orientation `réaliser-les-démarches-en-ligne` est précisé.
            """,
            examples=[
                "https://insersol.fr/biclou-atelier-reparation-participatif-solidaire/"
            ],
        ),
    ] = None
```